docs/install.md

```diff
@@ -1,5 +1,7 @@
 # Installation
 
+If PHP (>= 5.6) is not installed on your environment, install it now; PHP is a requirement for Dock CLI.
```

Comment:
Is this relevant for someone who is just installing the tool rather than contributing to it?